src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: "Useful Dart Libraries You Should Know About"
+description: "What are some of the most useful and popular Dart libraries and where can you learn more."
+---
+
+You can leverage a wide array of available Dart libraries and packages when
+writing your code. This document shares some of the most widely
+used libraries and library packages and tells you where you can learn more.
+
+<aside class="alert alert-info" markdown="1">
```

Comment:
Oh, reading further now I see the doc info down in the next section. I expected that section to have info _only_ about dart:xyz and xyz libraries.